src/app/components/AccountPage/AccountPage.jsx

```diff
@@ -20,63 +23,33 @@ const AccountPage = (props) => {
     payload: newContent,
   });
 
-  // Build object in the format used by Legacy Catalog
-  // [input[name]]: input[value]
-  const [selectedItems, updateSelectedItems] = useState({});
-
   const [errorMessage, updateErrorMessage] = useState(null);
+  const [isLoading, toggleLoadingState] = useState(true);
 
   useEffect(() => {
+    if (errorMessage) console.error(errorMessage);
     if (!patron.id) {
       const fullUrl = encodeURIComponent(window.location.href);
       window.location.replace(`${appConfig.loginUrl}?redirect_uri=${fullUrl}`);
     }
     const accountPageContent = document.getElementById('account-page-content');
 
-    if (!accountPageContent) return;
-    const links = accountPageContent.getElementsByTagName('A');
-    const checkboxes = accountPageContent.querySelectorAll('input[type=checkbox]');
-
-    // this "Ratings" feature is in the html, but is not in use
-    accountPageContent.getElementsByTagName('th').forEach((th) => {
-      if (th.textContent.includes('Ratings')) th.style.display = 'none';
-    });
-    accountPageContent.getElementsByClassName('patFuncRating').forEach((el) => { el.style.display = 'none'; });
-
-    const errorMessageEls = accountPageContent.getElementsByClassName('errormessage');
-    if (errorMessageEls.length) {
-      // in original HTML this is `hidden`
-      errorMessageEls[0].style.display = 'block';
+    if (accountPageContent && isLoading) {
```

Comment:
Yeah. I have to figure out a better way to use the hooks here. It is fine for the initial load. But we really need it when the buttons are used to prevent other requests from being made at the same time.